packages/stylelint-config-vue/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@convidera-team/stylelint-config-vue-convidera",
-  "version": "3.2.0",
+  "version": "3.2.1",
```

Comment:
if the version was changed by hand please revert it, the version of the package will be changed automatically when running `yarn lerna:version`